OpenRA.Game/Exts.cs

```diff
@@ -111,12 +111,6 @@ public static bool LinesIntersect(int2 a, int2 b, int2 c, int2 d)
 			return WindingDirectionTest(c, d, a) != WindingDirectionTest(c, d, b) && WindingDirectionTest(a, b, c) != WindingDirectionTest(a, b, d);
 		}
 
-		public static bool HasModifier(this Modifiers k, Modifiers mod)
-		{
-			// PERF: Enum.HasFlag is slower and requires allocations.
```

Comment:
This was fixed in .net core

Reply:
We still have the macos compat release and people self compiling with mono, but if this is only a ~couple of percent level regression for them then IMO lets get rid of it.